saleor/site/models.py

```diff
@@ -18,6 +18,8 @@
 
 patch_contrib_sites()
 
+DEFAULT_LIMIT_QUANTITY_PER_CHECKOUT: Final[int] = 50
```

Comment:
IMHO no, because the main part of this task was to get rid  of limits from settings and make it configurable via api. This is just a default value for model to give some "reasonable" value, that reflects former default settings setup.